machines/pod_X1Y1_ruche_X16Y8_hbm_one_pseudo_channel/Makefile.machine.include

```diff
@@ -71,7 +71,7 @@ BSG_MACHINE_VCACHE_WAY                = 8
 BSG_MACHINE_VCACHE_LINE_WORDS         = 16
 BSG_MACHINE_VCACHE_STRIPE_WORDS       = $(BSG_MACHINE_VCACHE_LINE_WORDS)
 BSG_MACHINE_VCACHE_WORD_TRACKING      = 1
-BSG_MACHINE_VCACHE_IPOLY_HASHING      = 0
+BSG_MACHINE_VCACHE_IPOLY_HASHING      = 1
```

Comment:
@tommydcjung Would you suggest enabling ipoly hashing by default in all non-bigblade machine files?

Reply:
I suggest keeping the 2x2 mesh as a machine with everything minimal (no ipoly, etc). We don’t want a cross-product, but it can be a machine that has most things “turned off”

Reply:
Currently, I-poly is implemented for a pod that is 16 or 32 tiles wide. I would enable it for pod_X1Y1_ruche_X16Y8_hbm_one_pseudo_channel, which is the baseline configuration.